Xamarin.Forms.Platform.Android/CollectionView/ObservableItemsSource.cs

```diff
@@ -7,9 +7,8 @@ namespace Xamarin.Forms.Platform.Android
 {
 	internal class ObservableItemsSource : IItemsViewSource
 	{
-		// TODO hartez 2018/07/30 14:40:11 We may need to implement IDisposable to make sure this all gets cleaned up	
-		readonly RecyclerView.Adapter _adapter;
-		readonly IList _itemsSource;
+		RecyclerView.Adapter _adapter;
```

Comment:
Since we don't need to `null` these out, they don't actually change and we can leave these as `readonly`.